cluster-autoscaler/Makefile

```diff
@@ -28,7 +30,7 @@ ifdef DOCKER_RM
 else
   RM_FLAG=
 endif
-IMAGE=$(REGISTRY)/cluster-autoscaler$(PROVIDER)
+IMAGE=$(REGISTRY)/cdk/cluster-autoscaler$(PROVIDER)
```

Comment:
In a conversation with Kevin, he mentioned these images should go under the cdk subdirectory in the registry.

Reply:
so, what does this turn out to be?
```
BUILD_TAGS=juju
upload.rocks.canonical.com:5000/cdk/cluster-autoscaler-juju
```

Reply:
sorry, replied to the wrong one down below. There will be images for each arch:
```
upload.rocks.canonical.com:5000/cdk/cluster-autoscaler-juju-amd64
upload.rocks.canonical.com:5000/cdk/cluster-autoscaler-juju-arm64
```
and we can add tags for CLI and nonCLI versions to end up with 4 different images:
```
upload.rocks.canonical.com:5000/cdk/cluster-autoscaler-juju-amd64:dev-CLI
upload.rocks.canonical.com:5000/cdk/cluster-autoscaler-juju-amd64:dev
upload.rocks.canonical.com:5000/cdk/cluster-autoscaler-juju-arm64:dev-CLI
upload.rocks.canonical.com:5000/cdk/cluster-autoscaler-juju-arm64:dev
```

Reply:
ok, cool.  I see how that lines up with the other builds.  You'll want numerical tags though.

```
dev-CLI-0
dev-CLI-1
dev-CLI-2
```